plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -836,8 +835,18 @@ def modify_xml(self):
         self._templatize_pkgid(other)
         self._templatize_pkgid(filelists)
 
+        text_other = utils.element_to_text(other)
+
+        other_l = text_other.find(">")
+        other_r = text_other.rfind("</")
```

Comment:
Can there be more than one ">" and "</" pair that need to be handled? If not, I'm fine with this solution. If so, then this gets a little more complicated, since we need to deal with the multiple instances and also ensure that we have the correct number and order of verbatim/endverbatim templatetags.